locales/en/common.json

```diff
@@ -4,7 +4,10 @@
         "subtitle": "We respect your inbox. No spam, promise ✌️",
         "nameInput": "Your name",
         "emailInput": "Your email",
-        "subscribeBtn": "Subscribe"
+        "subscribeBtn": "Subscribe",
+        "successTitle": "Thank you for subscribing!",
+        "errorTitle": "Something went wrong",
+        "errorSubtitle": "Please try again later."
```

Comment:
Thanks for adding these in :)
Could you please add translations for other locales as well that are present in `locales` folder.

The tool recommended by Maciej was https://www.deepl.com/translator for initial translations.

Reply:
Sure 👍